src/app/components/common/ImageDiv/ImageDiv.tsx

```diff
@@ -8,15 +8,22 @@ type ImageDivProps = {
     image: string;
     width: number;
     height: number;
-    classIdentifier?: string;
+    setOpenExplorerRefs?: Function;
 };
 
 const ImageDiv = (props: ImageDivProps) => {
-    const { onClick, image, width, height } = props;
+    const { onClick, image, width, height, setOpenExplorerRefs } = props;
     const classes = useStyles({ image, width, height });
+    const ref = useRef(null);
+
+    useEffect(() => {
+        if (setOpenExplorerRefs) {
+            setOpenExplorerRefs((state: any) => [...state, ref.current]);
+            }
+        }, []);
```

Comment:
Code block should be properly formatted (normalized block spacing)

Reply:
done